src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -268,14 +268,32 @@ private static OperatorKind GetOperatorKind(IMethodSymbol method)
         /// </summary>
         public SyntaxNode ParameterDeclaration(IParameterSymbol symbol, SyntaxNode? initializer = null)
         {
-            return ParameterDeclaration(
+            var parameter = ParameterDeclaration(
```

Comment:
if you're interested, it would be valuable to see if we can replace this with a call to the ParameterGenerator that ICodeGenerationService uses.